classes/crucible.php

```diff
@@ -224,28 +257,30 @@ public function getall_attempts($state = 'all', $review = false) {
 
         if ((!$review) || (!$this->is_instructor())) {
             debugging("getall_attempts for user", DEBUG_DEVELOPER);
-            $where[] = 'userid = ?';
-            $sqlparams[] = $USER->id;
+            $where[] = '({crucible_attempts}.userid = ? OR {crucible_attempt_users}.userid = ?)';
+            $sqlparams[] = $user;
+            $sqlparams[] = $user;
```

Comment:
for emphasis

Reply:
that's actually how you add something to the end of an array in php, apparently!